cli/args/flags.rs

```diff
@@ -4165,6 +4170,7 @@ fn bundle_parse(flags: &mut Flags, matches: &mut ArgMatches) {
 }
 
 fn cache_parse(flags: &mut Flags, matches: &mut ArgMatches) {
+  log::info!("{}", colors::yellow("⚠️ Warning: `deno cache` is deprecated and will be removed in Deno 2.1.\nUse `deno install` instead."));
```

Comment:
This has side-effects for flag parsing - which is an anti-pattern. Can you move this warning to the `main.rs` file where `deno cache` is actually invoked?